recipes/pybind11/all/conanfile.py

```diff
@@ -20,7 +20,8 @@ class PyBind11Conan(ConanFile):
 
     def source(self):
         tools.get(**self.conan_data["sources"][self.version])
-        os.rename("{}-{}".format(self.name, self.version), self._source_subfolder)
+        os.rename("{}-{}".format(self.name, self.version),
+                  self._source_subfolder)
```

Comment:
Could you restore all these changes based on length? CCI's .editorconfig allows 200 characters per line.

Reply:
In the [PR template](https://github.com/conan-io/conan-center-index/blob/master/.github/PULL_REQUEST_TEMPLATE.md) I had to tick off "I've followed the PEP8 style guides for Python code in the recipes." In [PEP8](https://www.python.org/dev/peps/pep-0008/#maximum-line-length) it says 79 chars per line (and optionally up to 99 chars). I started using autopep8 in order to comply to this rule.
Do we have to change the PR template to reflect this? With which tool do I use an editorconfig file?

Reply:
Line length is indeed an exception to the rest of PEP8 style

See https://editorconfig.org for a list of editors with native support and a list of plugins for even more editors